### PR TITLE
This is my first user story for sprint 3- list only children

### PR DESCRIPTION
**Pull request by samanthainneo99**
This is my first user story for sprint 3- us46- list only children.

### CODE REVIEW

**Review by mmccree1 — APPROVED** [review]

File: src/us12.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+Sprint:     Sprint 3
+Use Case:   Checks if parents are too old to have children
+'''
+
+import pandas as pd
+from datetime import date
+import copy
+import os
+from Gedcom import Gedcom, Family
+from us28 import sibList
```
Comment (ming1in) [inline src/us12.py:12]: whats sibList? you should just be able to do something like `from us28 import us28`

File: test/seeds/test124.ged [hunk 2]
```diff
@@ -0,0 +1,196 @@
+0 HEAD
```
Comment (ming1in) [inline test/seeds/test124.ged]: should put all your test files into their own folder within seeds

File: test/us12_test.py [hunk 3]
```diff
@@ -0,0 +1,34 @@
+import unittest
+import datetime
+import sys
+import os
+sys.path.append(os.path.abspath('../src'))
+import us12
```
Comment (ming1in) [inline test/us12_test.py]: should be able to do` from us12 import us12`

File: test/us12_test.py [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+class us12_test(unittest.TestCase):
+    def test1(self):
+        filename = "seeds/test121.ged" 
+        ret = ['I2']
+        self.assertEqual(us12.us12(filename),ret)
```
Comment (ming1in) [inline test/us12_test.py]: if you use the import in the previous comment ya could just do `us12()` instead of `us12.us12()` just a neat trick

**Review by ming1in — APPROVED** [review]
some small comments were left, but good overall. before you merge:

- add test to `test_suite.py`
- put your test's seed files into their own folder in /seeds

**Review by arana23 — APPROVED** [review]
Nice Job! :)

**Pull request merged**